lib/encoder.js

```diff
@@ -211,6 +212,9 @@ class Encoder extends stream.Transform {
   }
 
   _pushIntNum(obj) {
+    if (obj === 0 && 1/obj < 0)
```

Comment:
Maybe use `Object.is(obj, -0)`?

Reply:
Yea, that would be more self explanatory.    Thanks.